mutation_partition_view.cc

```diff
@@ -273,7 +274,7 @@ future<> mutation_partition_view::do_accept_gently(const column_mapping& cm, Vis
     }
 }
 
-template <bool is_preemptible>
+template <bool is_preemptible, consume_in_reverse reverse>
```

Comment:
I won't ask to redo the series, but splitting it by first adding the parameters everywhere, then adding the contents, isn't helping review. For example, how does the reviewer know you haven't forgotten an implementation somewhere?

Patches shouldn't add bugs, and this one does: an API that accepts an instruction to reverse, then ignores it.

Reply:
I will at least add a comment explaining it.